pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "heal-sdk"
-version = "1.0.1"
+version = "1.1.1"
```

Comment:
```suggestion
version = "1.1.0"
```

Reply:
Right, version has been updated, thank you.